docs/server/guides/setup/minecraft-setup.md

```diff
@@ -164,9 +165,9 @@ This will automatically take you inside the tmux session. Start the server using
 java -Xmx1024M -Xms1024M -jar server.jar nogui
 ```
 
-Exit out of the tmux session using ++ctrl+b++ **then** ++d++
+Exit out of the tmux session using `ctrl + b` **then** `d` 
```

Comment:
Please keep this as it was. This is the syntax used in Mkdocs for formatting keys.

Take a look here: https://squidfunk.github.io/mkdocs-material/reference/formatting/#adding-keyboard-keys

Reply:
kk